plugins/discovery-file/build.gradle

```diff
@@ -26,12 +26,6 @@ esplugin {
     classname 'org.elasticsearch.discovery.file.FileBasedDiscoveryPlugin'
 }
 
-bundlePlugin {
-    from('config/discovery-file') {
-        into 'config'
-    }
-}
-
 task setupSeedNodeAndUnicastHostsFile(type: DefaultTask) {
```

Comment:
These integration tests remain in the plugin, to be removed at a later date. Since we plan on using file-based discovery for most of the cluster integration tests there seems little value in moving this.